htlcswitch/switch_test.go

```diff
@@ -4346,6 +4346,9 @@ func TestSwitchDustForwarding(t *testing.T) {
 				totalDust += localMailDust
 			}
 
+			t.Logf("totalDust=%v, expectedDust=%v", totalDust,
```

Comment:
nit: could use `wait.NoError()` instead of `for` loop with poll interval and timeout, and return expected value as an error if not reached in time.

Reply:
Done!